src/test/java/Tests/UpdateProfileTests.java

```diff
@@ -0,0 +1,212 @@
+			Thread.sleep(6000);
+}
+	@Test(priority=10, description = "Verifying Updating College with Invalid Year")
+	public void VerifyUpdatingCollegeDetailsWithInvalidYear() throws Exception{
+		userProfilePage.Updatecollege("Indian Institute of Information Technology, Sonipat ");
+		userProfilePage.UpdateJoiningYear("abc2019");
+		userProfilePage.UpdateEndingYear("abc2001");
+		Thread.sleep(1000);
+		Assert.assertEquals(userProfilePage.getErrorMessageValidYear(), "Please enter a valid year" );
+		
+	}
+	@Test(priority=11, description = "Verifying Updating College with Valid Details")
+	public void VerifyUpdatingCollegeDetails()throws Exception{
+		userProfilePage.Updatecollege("Indian Institute of Information Technology, Sonipat");
+	    userProfilePage.UpdateJoiningYear("2012");
+		userProfilePage.UpdateEndingYear("2016");
+		userProfilePage.saveButton.click();
+		Thread.sleep(2000);
+		Assert.assertEquals(userProfilePage.getUpdateMessageforEducationDetails(), "User Profile updated" );
+		Thread.sleep(6000);
+			}
+	
+	
+	@Test(priority=12, description = "Verifying Updating Personal Details with Valid Details")
+	public void VerifyUpdatingPersonalDetails()throws Exception{
+		userProfilePage.btnEditDetails.click();
+		Thread.sleep(1000);
+		userProfilePage.UpdatePersonalDetails("TestFirstName", "TestLastName", "9810101010", "Test Address",
+		"India", "National Capital Territory of Delhi", "Delhi", "110091");
+	    userProfilePage.updateDob();
+		userProfilePage.btnSave.click();
+		Thread.sleep(1000);
+		Assert.assertEquals(userProfilePage.getUpdateMessageforPersonalDetails(), "User Profile updated" );
+	} 
+	
+	@Test(priority=13, description = "verify update share performance with no one")
+	public  void VerifyUpdatingShareMyPerformanceDetailsWithNoOne()throws Exception{
+		userProfilePage.sharePerfWithNoOne.click();
+		Thread.sleep(500);
+		Assert.assertEquals(userProfilePage.getUpdateMessageforsharingPerformanceWith(), "User Profile updated" );
+		}
+	
+	@Test(priority=14, description = "verify update share performance with My Friends")
+	public  void VerifyUpdatingShareMyPerformanceDetailsWithMyFriends()throws Exception{
+		userProfilePage.sharePerfWithFriends.click();
+		Thread.sleep(500);
+		Assert.assertEquals(userProfilePage.getUpdateMessageforsharingPerformanceWith(), "User Profile updated" );
+		}
+	
+	@Test(priority=15, description = "verify update share performance with All User's")
+	public  void VerifyUpdatingShareMyPerformanceDetailsWithAllUsers()throws Exception{
+		userProfilePage.sharePerfWithAllUser.click();
+		Thread.sleep(500);
+		Assert.assertEquals(userProfilePage.getUpdateMessageforsharingPerformanceWith(), "User Profile updated" );
+		}
+	
+	@Test(priority=16, description = "Verifying Update Password with Incorrect Current Password")
+	public void  verifyingupdatepasswordinvalidcurrentpassword()throws Exception{
+		Thread.sleep(1000);
+		userProfilePage.currentPassword.sendKeys("incorrectcurrentpassword");
```

Comment:
create one common method for updatepassword and use it here

Reply:
Created method :- incorrectCurrentPassword().